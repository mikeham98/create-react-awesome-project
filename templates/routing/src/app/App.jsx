import React from 'react';
import Routes from './routes/';

export default class App extends React.Component {
    render() {
        return (
            <div>
                <Routes />
            </div>
        );
    }
}