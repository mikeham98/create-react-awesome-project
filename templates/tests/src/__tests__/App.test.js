import React from 'react';
import {shallow} from 'enzyme';
import App from '../app/App';

describe('App', () => {
    let props = {};
    let shallowedWrapper;
    let wrapper = () => {
        if(!shallowedWrapper) {
            shallowedWrapper = shallow(<App {...props}/>);
        }
        return shallowedWrapper;
    };

    beforeEach(() => {
        props = {
            
        };
        shallowedWrapper = undefined;
    });
    
    it('should render app', () => {
        expect(wrapper()).toMatchSnapshot();
    });
});