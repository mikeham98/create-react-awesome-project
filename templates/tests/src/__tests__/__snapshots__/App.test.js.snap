// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render app 1`] = `
<div>
  Hello world lol
</div>
`;
